### ISSUE TITLE
Provide Polybar as an optional package

### ISSUE THREAD

**Issue by drewofdoom** [regolith-linux/regolith-desktop#50]
While I think that providing i3bar/i3blocks as the default is the best choice for simplicity, I have found that Polybar provides a nicer experience overall for me. I do see that you have played with Polybar in previous iterations. Any chance you would package it up for Ubuntu and offer it in the Regolith PPA? The current installation method that I could find is from source, which is non-ideal for the obvious reasons.

**Comment by MoreThanLuck** [comment]
I'd also like to see this!

**Comment by kgilmer** [comment]
I would like to provide this and also a general-use packaging of i3-gaps.  I would like to do it in such a way that others can easily just fork the repo and publish updates to their own PPAs in the case that I fall behind versions or others want newer versions before I can get to them.

If anyone has other packages that they'd like to be able to install without dealing w/ the source code let me know and if it's straight forward I'll add them as well.

**Comment by MoreThanLuck** [comment]
I'd love to see Polybar support added!

**Comment by kgilmer** [comment]
Here it is!

https://github.com/regolith-linux/speed-ricer

**Comment by pedrud** [comment]
I wasn't sure where to put this, so we'll give it a go here. First off, great work combining Ubuntu and i3. I've used a variety of distros over the years and always come back to Ubuntu for stability. I went about it a slightly different way and installed a minimal version of 19.10 and added the Regolith desktop on top (I recently started trying out i3 after using XFCE for years). Anyhow, I digress.

I also installed and got polybar working and configured before I discovered the speed-ricer. Ricing is not something I plan on doing, but I like what polybar offers. 

As a new i3 user and specifically on Reglith, I cannot figure out where to disable the i3bar and put my `exec_always --no-startup-id $HOME/.config/polybar/launch.sh` code to start polybar on login. Thanks.

**Comment by kgilmer** [comment]
Hi @pedrud :smile: , I have just created a HowTo that covers switching to Polybar from i3bar: https://github.com/regolith-linux/regolith-desktop/wiki/HowTo:-Swap-out-i3bar-for-Polybar

Regarding the ricing, it's just an attempt at a clever name.  The project simply offers packages that are not available in Ubuntu's standard package repositories that are useful for desktop customization.

In your case specifically, you can determine which i3 config file is being used like this:
```bash
$ i3 --moreversion
```

**Comment by pedrud** [comment]
@kgilmer Cheers! I will take a look at that. I did end up getting some help from the people over at Polybar. Thanks for the ricing explanation.

Thanks for the i3 tip. Between the suggestions over at polybar and reading your configuration docs, I ended up getting it working. As a newbie to i3 and polybar, I will take a look at your HowTo and see if it make sens from that perspective and offer any suggestions if needed. Thanks again. I'm really liking Regolith! Brilliant job.
